workbench/src/main/setupOpenExternalUrl.js

```diff
@@ -8,7 +8,6 @@ import { ipcMainChannels } from './ipcMainChannels';
 export default function setupOpenExternalUrl() {
   ipcMain.on(
     ipcMainChannels.OPEN_EXTERNAL_URL, (event, url) => {
-      event.preventDefault();
```

Comment:
This doesn't belong. This `event` is a node `Event` instance, not a browser `Event`. We already did `event.preventDefault()` on the browser event earlier in the renderer right before sending the IPC here.